Sources/MessagingInApp/Gist/Network/BaseNetwork.swift

```diff
@@ -31,7 +32,7 @@ enum BaseNetwork {
         URLSession.shared.dataTask(with: urlRequest, completionHandler: { data, response, error in
             if let error = error { completionHandler(.failure(error)) }
             guard let data = data, let response = response as? HTTPURLResponse,
-                  (200 ... 299).contains(response.statusCode)
+                  (200 ... 304).contains(response.statusCode)
```

Comment:
`304` is content not modified, I know we were relying on it before but were returning `failure` apparently, so just wanted to check nothing breaks from before right?

Reply:
Before this change, we used to rely on the default `NSURLSession`'s behavior to cache any 304 responses, it used to take 304s and convert them to the latest 200 responses automatically, we can't have that anymore because otherwise, the new header would be part of that cached response.